src/pages/Vote/index.tsx

```diff
@@ -122,136 +124,140 @@ export default function Vote() {
 
   return (
     <>
-      <PageWrapper gap="lg" justify="center">
-        <DelegateModal
-          isOpen={showDelegateModal}
-          onDismiss={toggleDelegateModal}
-          title={showUnlockVoting ? <Trans>Unlock Votes</Trans> : <Trans>Update Delegation</Trans>}
-        />
-        <TopSection gap="md">
-          <VoteCard>
-            <CardBGImage />
-            <CardNoise />
-            <CardSection>
-              <AutoColumn gap="md">
-                <RowBetween>
-                  <TYPE.white fontWeight={600}>
-                    <Trans>Uniswap Governance</Trans>
-                  </TYPE.white>
-                </RowBetween>
-                <RowBetween>
-                  <TYPE.white fontSize={14}>
+      <Route exact strict path="/vote/:governorIndex/:id" component={VotePage} />
+      <Route exact strict path="/vote/create-proposal" component={CreateProposal} />
+      <Route exact strict path="/vote">
```

Comment:
I'm adding a `<Route />` here so those elements won't get rendered for other routes like `/vote/1/2` or `/vote/create-proposal`.